src/main/java/io/github/syst3ms/skriptparser/expressions/ExprLength.java

```diff
@@ -0,0 +1,40 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.PropertyExpression;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.Function;
+
+/**
+ * Length of a string.
+ *
+ * @name Length
+ * @pattern length of %string%
+ * @pattern %string%'s length
+ * @since ALPHA
+ * @author Romitou
+ */
+public class ExprLength extends PropertyExpression<Number, String> {
+
+    static {
+        Main.getMainRegistration().addPropertyExpression(
+                ExprLength.class,
+                Number.class,
+                true,
+                "string",
+                "length"
+        );
+    }
+
+    @Override
+    public Function<String[], Number[]> getPropertyFunction() {
+        return strings -> new Number[] {strings[0].length()};
```

Comment:
A alright, the last change I'll request is simply some ""null safety"". By that I mean that `getValues()` (which is fed into this function) may be empty.
I'll say again the fundamental equivalence in this parser : `getSingle() == null` implies that `getValues().length == 0` and vice-versa.

The function should just check if the array is null and return something like -1 in that case.

Reply:
All right, thank you!